bigquery_etl/cli/query.py

```diff
@@ -0,0 +1,144 @@
+"""bigquery-etl CLI query command."""
+
+import click
+import os
+from pathlib import Path
+import re
+import sys
+import string
+
+from bigquery_etl.metadata.parse_metadata import Metadata
+
+
+QUERY_NAME_RE = re.compile(
+    r"(?P<dataset>[a-zA-z0-9_]+)\.(?P<name>[a-zA-z0-9_]+)(?P<ver>_v[0-9]+)?"
+)
+
+
+def create_query_cli_group():
+    """Create the CLI group for the query command."""
+    group = click.Group(name="query", help="Creating/validating/deleting/... queries")
```

Comment:
```suggestion
    group = click.Group(name="query", help="Commands for manipulating queries")
```

Reply:
I think the idiomatic way of exposing this is [through the decorators](https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts). It'll look a little less crowded that way. In this file, it would look something like the following:

```python
@click.group()
def query():
    pass

@query.command()
@click.option(...)
...
def create(...):
    # logic for creating a query
    pass
```